eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -223,6 +225,78 @@ spec =
           enactState <- getsNES $ newEpochStateGovStateL . cgEnactStateL
           rsEnactState pulserRatifyState `shouldBe` enactState
 
+      xit "policy is respected by proposals" $ do
+        let
+          scriptHash = hashScript @era $ alwaysSucceeds @'PlutusV2 0
+          wrongScriptHash = hashScript @era $ alwaysSucceeds @'PlutusV2 1
```

Comment:
PlutusV2 can't be used in governance policy :wink: 

Same as with the other comment. I'll fix that.